influxql/parser_test.go

```diff
@@ -43,6 +43,26 @@ func TestParser_ParseQuery_Empty(t *testing.T) {
 	}
 }
 
+// Ensure the parser will skip comments.
+func TestParser_ParseQuery_SkipComments(t *testing.T) {
```

Comment:
There are three more specific test cases I'd like to see covered:

1. `-- some comment containing /*` is treated as a line comment, not a block comment (I think that behavior is already true, just looking for test coverage)
2. Block comment inside a statement, e.g. `SELECT user /*, system, idle */ FROM cpu` is equivalent to `SELECT user FROM cpu`
3. A well-formed regex looks like it contains a block comment but remains a regex, e.g. `SELECT value FROM /foo\/*bar/ WHERE x = 1`, and maybe `/foo\/*bar*/` as well